Dockerfile_mbtci_template

```diff
@@ -12,9 +12,19 @@ ARG SAPMACHINE_VERSION=JAVA_VERSION_TEMPLATE
 
 # Environment variables
 ENV PYTHON /usr/bin/python3
+
 ENV MAVEN_HOME /usr/share/maven
 ENV M2_HOME ${MAVEN_HOME}
-ENV PATH /usr/local/go/bin:$PATH
+ENV JAVA_HOME /opt/jdk
+ENV PATH $PATH:$MAVEN_HOME/bin:${JAVA_HOME}/bin
+
+ENV NODE_HOME /usr/local/nodejs
+ENV PATH $PATH:${NODE_HOME}/bin
+
+ENV GOPATH /go
+ENV GO_HOME /usr/local/go
+ENV PATH $GOPATH/bin:$GO_HOME/bin:$PATH
```

Comment:
The GNU/Linux file hierarchy standard (aka FHS) does not include all that paths. That's why Dockerfile usually creates symlinks to FHS paths instead of playing with $PATH env variable to avoid introducing security issues. 

Defining the installation path for Go, Node.js, Maven is fine (but not strictly needed) and is out of that PR scope and shall be done in another PR. And given the extra commands added to cope with them that introduce permission issues, not thought enough to be part of that PR. Please revert. That PR should only focus on integrating the java 8 and nothing more. The original PR was doing exactly that, please just take it as it was, do any minor fixes to it found during our tests and merge it.

Reply:
I will revert the GO_HOME env, and when install go, use GO_HOME as a local var